README.md

```diff
@@ -10,45 +10,62 @@ This bot is meant to support d.craft in creating, managing, and hosting tourname
+`!ign unlink`: Attempts to unlink you to any IGN they may have linked before. Only succeeds if you already have a linked IGN.
 
 `!ign list`: Lists all discord users that have linked their IGN to their discord.
 
+### Teams Commands
+
+These commands are meant to allow users to create, modify, and delete teams, while being as simple and efficient as possible.
+
+`!team create <tag> "name" "description"`: Creates a team with the information given. Tags can be 1-6 letters long and alphanumeric; descriptions can be up to 255 characters long.
+
+`!team info [tag]`: Grabs the information about a team, such as its leader, its full name with tag, and its description. If no tag is specified, the command grabs the information about the team you are currently in.
+
+`!team invite <@mention>`: Invites a player to your team. Only works if you are the leader of the team.
+
+`!team join <tag>`: Joins a team with the tag specified. Only works if you already have an invite to that team.
+
+`!team kick <@mention>`: Forcibly removes a player from your team. Only works if you are the leader of the team.
+
+`!team delete`: Deletes the team alltogether, including the roles and channel. Only works if:
+
+- You are the leader of the team
+- There are no other members in the team
+
+`!team transfer <@mention>`: Transfers ownership of the team to the person mentioned. Only works if you are the leader of the team.
+
+`!team edit <name | description> "value"`: Edits the team's name or description. Only works if you are the leader of the team.
+
+`!team leave`: Leaves the team you're currently in. Only works if you are not the leader of the team.
+
 ## Modules Used
 
-| Module     | npm Link                                         | Version | Purpose                                                                 |
-| ---------- | ------------------------------------------------ | ------- | ----------------------------------------------------------------------- |
-| Discord.js | [Link](https://www.npmjs.com/package/discord.js) | v12.5.1 | Connect to Discord and send messages, get data from Discord users, etc. |
-| Axios      | [Link](https://www.npmjs.com/package/axios)      | v0.21.1 | Generate API requests and get information from Hypixel                  |
-| Chalk      | [Link](https://www.npmjs.com/package/chalk)      | v4.1.0 | Make colored console messages                  |
-| Knex       | [Link](https://www.npmjs.com/package/knex)      | v0.21.15 | Create and modify SQL tables                   |
-| PG         | [Link](https://www.npmjs.com/package/pg)      | v8.5.1 | Knex dependency                  |
-| Nodemon    | [Link](https://www.npmjs.com/package/nodemon)      | v2.0.4 | Automatically restart bot whenever changes are made                  |
+| Module     | npm Link                                         | Version  | Purpose                                                                 |
```

Comment:
This section doesn't make that much sense to keep, but I don't mind if you like it :)

Reply:
Yeah I was thinking the same I'll delete it